src/main/less/core.less

```diff
@@ -279,3 +279,15 @@ svg:not(:root) {
         }
     }
 }
+
+.no-data-text {
+    font-family: "Helvetica Neue";
+    font-size: 24px;
+    text-anchor: middle;
+    fill: #868a8c;
```

Comment:
I see that this is a color recommendation from UX from [here](https://jira3.cerner.com/browse/CARBON-52?focusedCommentId=6929787&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-6929787)
But instead of using it like this (as a hex), lets create a color variable in `colors.less` along with others and use that. All colors are standard ones picked from [here](https://wiki.cerner.com/pages/viewpage.action?spaceKey=UserExperience&title=Color+Palette).

Reply:
Taken color from color.less which is defined in wiki